data/settings.py

```diff
@@ -10,7 +10,7 @@
 # See https://docs.djangoproject.com/en/1.9/howto/deployment/checklist/
 
 # SECURITY WARNING: don't run with debug turned on in production!
-DEBUG = bool(os.getenv("DEBUG", False))
+DEBUG = True if (os.getenv('DEBUG') == 'true') else False
```

Comment:
Good point. Changing just case with any string someone can make an honest mistake of doing ``DEBUG=false`` instead of ``unset DEBUG``